typo fix

setanarut opened the pull request:
<!--
Pull requests should always be made for the `master` branch first, as that's
where development happens and the source of all future stable release branches.

Relevant fixes are cherry-picked for stable branches as needed.

Do not create a pull request for stable branches unless the change is already
available in the `master` branch and it cannot be easily cherry-picked.
Alternatively, if the change is only relevant for that branch (e.g. rendering
fixes for the 3.2 branch).
-->

setanarut committed:
Missing brackets and function arguments fix

fixes;
Parse Error: Too few arguments for "tween_property()" call. Expected at least 4 but received 3.

setanarut committed:
Merge pull request #1 from hazarek/hazarek-patch-1

Missing brackets and function arguments fix

setanarut committed:
Merge branch 'master' into master

KoBeWi commented on doc/classes/Tween.xml:
```diff
@@ -267,16 +267,16 @@
 				Creates and appends a [PropertyTweener]. This method tweens a [code]property[/code] of an [code]object[/code] between an initial value and [code]final_val[/code] in a span of time equal to [code]duration[/code], in seconds. The initial value by default is a value at the time the tweening of the [PropertyTweener] start. For example:
```
```suggestion
				Creates and appends a [PropertyTweener]. This method tweens a [code]property[/code] of an [code]object[/code] between an initial value and [code]final_val[/code] in a span of time equal to [code]duration[/code], in seconds. The initial value by default is the value at the time the tweening of the [PropertyTweener] starts. For example:
```
Another typo.

setanarut committed:
Missing square brackets fix.

var a = PackedColorArray(0, 0, 0, 1, 1, 1, 1, 1) INVALID
var a = PackedColorArray([0, 0, 0, 1, 1, 1, 1, 1]) VALID
var b = PackedColorArray([Color.WHITE, Color.BLACK]) I preferred the legible one.

setanarut committed:
Merge pull request #2 from hazarek/hazarek-typo-fix

Missing square brackets fix.

Chaosus commented:
@hazarek You need to squash the commits as required by our pipeline (see https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html).

hazarek commented:
> @hazarek You need to squash the commits as required by our pipeline (see https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html).

I'm completely inexperienced, can you fix this and make a pull request? Can you fix them all?
https://github.com/godotengine/godot/pull/58897,
https://github.com/godotengine/godot/pull/58764
